tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py

```diff
@@ -0,0 +1,60 @@
+import ctypes
+import time
+import argparse
+
+def Classify(model_path, text, input_tensor_name, output_score_tensor_name):
```

Comment:
python function should be named in snake_case rather than PascalCase (classify here).

Reply:
Hello Zhang, I have changed the method name to 'classify'